docs/integrations/microsoft-azure/azure-storage.md

```diff
@@ -81,6 +124,123 @@ In this section, you will configure a pipeline for shipping diagnostic logs from
    * Select `allLogs`.
    * Use the Event hub namespace and Event hub name configured in previous step in destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
 
+#### Activity Logs
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit).
+
+
+## Installing the Azure Storage app
+
+Now that you have set up data collection, install the Azure Storage Sumo Logic app to use the pre-configured [dashboards](#viewing-the-azure-storage-app-dashboards) that provide visibility into your environment for real-time analysis of overall usage.
+
+
+import AppInstallNoDataSourceV2 from '../../reuse/apps/app-install-index-apps-v2.md';
+
+<AppInstallNoDataSourceV2/>
+
+## Upgrading the Azure Storage app (optional)
+
+import AppUpdate from '../../reuse/apps/app-update.md';
+
+<AppUpdate/>
+
+## Uninstalling the Azure Storage app (optional)
+
+import AppUninstall from '../../reuse/apps/app-uninstall.md';
+
+<AppUninstall/>
+
+## Viewing the Azure Storage app dashboards
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+We highly recommend viewing these dashboards in the [Azure Storage view](/docs/dashboards/explore-view/#aws-observability) of the AWS Observability solution.
+
+### Azure Storage - Overview
+
+The **Azure Storage Overview** dashboard provides insights into transactions by storage account, storage type and api name.
```

Comment:
<dashboard title>

<dashboard description>

Use this dashboard to:
- <use case 1>
- <use case 2>

Refer https://help.sumologic.com/docs/integrations/amazon-aws/lambda/#request-analysis